configure.ac

```diff
@@ -235,6 +235,13 @@ AC_ARG_ENABLE([memcpy],
     [AC_HELP_STRING([--enable-memcpy],
                     [Use memcpy to perform local put/get operations (default:disabled)])])
 
+AC_ARG_ENABLE([fence],
+    [AC_HELP_STRING([--enable-fence],
+                    [enable and use FI_FENCE instead of transport layer fence in the OFI. (default: disabled)])])
```

Comment:
We are starting to use the "ofi" prefix for options tied to the OFI transport, so please use "ofi-fence" for this option. 
 Suggest this wording: "Use FI_FENCE feature to optimize put-with-signal operations. (default: disabled)"

Reply:
Looks good apart from this minor nitpick.  👍